app/views/groups/index.html.erb

```diff
@@ -0,0 +1,21 @@
+    <% if user_signed_in? %>
+      <%= render 'partials/navbar'  %>
+    <% end  %>
+
+<div class="categories" style="width:100%">
+  <div class="category_card_total px-4">
+   <p> Total: </p>
+    <p> <strong> <%= @total_amount %> </strong>  </p>
+ </div>
+  <div id="groups">
+    <% @groups.each do |group| %>
+      <%= link_to(user_group_entities_path(current_user, group), class:"text-decoration-none text-reset") do %>
+        <%= render group %>
+      <% end %>
+    <% end %>
+  </div>
+
+
+  <%= link_to "ADD CATEGORY", new_user_group_path(current_user), class:"mx-auto p-4 btn w-100 text-light btn_submit" %>
+</div>
```

Comment:
- [ ] Kindly follow the provided design (width, font, size, height, background, color...)  and make sure you add this white border around the Add Category button text. 👍🏻 The `New Transaction` button has a similar problem. Kindly fix it as well. 💯  👍🏻

- Original
![image](https://user-images.githubusercontent.com/79058364/230731290-6f44f560-b369-4d7e-9e4b-51130a17508b.png)

- Your Button
![image](https://user-images.githubusercontent.com/79058364/230731159-54b4dfeb-7147-4435-b66b-cad0835ba595.png)